app/views/shared/_header.html.haml

```diff
@@ -120,3 +120,6 @@
             %li= link_to t(:account_settings), edit_person_path(current_user), :class => 'profile_link'
             %li.divider{role: "separator"}
             %li.signout= link_to t(:sign_out), logout_path
+  #initiative.navtabs
+    - if @site && @site.id == Site.default.id
+      = image_tag "https://static.inaturalist.org/wiki_page_attachments/1249-original.png", width: 179
```

Comment:
Please check this image in to the repo and don't rely on a wiki page attachment that could change or disappear.